SimCivil.Concept/ItemModel/Composition/CompoundData.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace SimCivil.Concept.ItemModel
+{
+    /// <summary>
+    /// This file will be generated by data model
+    /// </summary>
+    public enum CompoundType
+    {
+        None,
+        Hydrogen,
+        Oxygen,
+        Water,
+        Wood,
+        Iron,
+    }
+
+    public struct CompoundProperty
+    {
+        public double Density;
+        public double MeltPoint;
+        public double BoilPoint;
+        public double FlashPoint;
+        public IReadOnlyDictionary<Element, double> Elements;
+    }
+
+    public class CompoundData
```

Comment:
应该实现ICompoundRepository
```suggestion
    public class JsonCompoundRepository : ICompoundRepository
```